src/host.js

```diff
@@ -10,9 +10,12 @@ export default class Host extends Port {
 			throw new Error(`Unable to extract origin from "${src}"`);
 		}
 
-		var parent = document.getElementById(id);
-		if(parent === null) {
-			throw new Error(`Could not find parent node with id "${id}"`);
+		var parent = id;
+		if (typeof id === 'string') {
```

Comment:
Suggestion: an alternative solution would be to update the constructor to accept a target element provider, thereby removing this logic from this constructor.  Another option would be to utilize named parameters to eliminate the assumption that if `id` is not a string, that it would be a DOM node.  This would also eliminate the type check.  

I personally prefer the provider solution because it helps reduce the host constructor to be more single purpose, and it becomes more extensible.